core/src/components/searchbar/searchbar.ionic.scss

```diff
@@ -49,25 +60,101 @@
 // -----------------------------------------
 
 .searchbar-clear-button {
-  @include globals.position(50%, globals.$ionic-space-200, null, null);
+  @include globals.position(
+    searchbarVars.$searchbar-ionic-input-clear-icon-size,
+    searchbarVars.$searchbar-ionic-input-clear-icon-size,
+    null,
+    null
+  );
 
-  position: absolute;
+  width: searchbarVars.$searchbar-ionic-input-clear-icon-size;
+  height: searchbarVars.$searchbar-ionic-input-clear-icon-size;
+
+  background-color: transparent;
 
-  width: globals.$ionic-scale-600;
-  height: globals.$ionic-scale-600;
+  font-size: globals.$ionic-font-size-400;
+
+  contain: strict;
+}
+
+// Searchbar Cancel Icon
+// -----------------------------------------
 
-  transform: translateY(-50%);
+.searchbar-cancel-button {
+  /**
+   * The left edge of the cancel button
+   * should align with the left edge
+   * of the back button if the searchbar
+   * is used in a toolbar.
+   */
+  @include globals.position(0, null, null, 9px);
```

Comment:
Would it make sense to create a global variable that would be used on the back button and wherever needed? As a TODO, not on this task probably

Reply:
I don't think so since these values are only being used in `searchbar`. If there are more components that share the values then we can come back to this conversation.